beetsplug/chroma.py

```diff
@@ -25,8 +25,8 @@
 import confuse
 
 from beets import config, ui, util
-from beets.autotag.distance import Distance
-from beets.metadata_plugins import MetadataSourcePlugin, TrackInfo
+from beets.autotag import Distance, TrackInfo
```

Comment:
Maybe we could import these directly from where they're defined?

```suggestion
from beets.autotag.distance import Distance
from beets.autotag.hooks import TrackInfo
```

Reply:
All I want long-term is to empty `autotag/__init__.py` completely 😅